#! /bin/sh /usr/share/dpatch/dpatch-run
## 09-init-lsb.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make initscript lsb cmpliant.

@DPATCH@
diff -urNad virtualbox-ose~/src/VBox/Installer/linux/vboxnet.sh virtualbox-ose/src/VBox/Installer/linux/vboxnet.sh
--- virtualbox-ose~/src/VBox/Installer/linux/vboxnet.sh	2008-02-20 14:47:20.000000000 +0100
+++ virtualbox-ose/src/VBox/Installer/linux/vboxnet.sh	2008-02-21 00:57:47.000000000 +0100
@@ -19,10 +19,10 @@
 #
 ### BEGIN INIT INFO
 # Provides:       vboxnet
-# Required-Start: $network
-# Required-Stop:
-# Default-Start:  3 5
-# Default-Stop:
+# Required-Start: $remote_fs $network
+# Required-Stop:  $remote_fs
+# Default-Start:  2 3 4 5
+# Default-Stop:   0 1 6
 # Description:    VirtualBox permanent host networking setup
 ### END INIT INFO
 
@@ -101,17 +101,19 @@
 fi
 
 if [ "$system" = "other" ]; then
+    . /lib/lsb/init-functions
+
     fail_msg() {
     	[ -f "$VARFILE" ] && rm $VARFILE
-        echo " ...fail!"
+        log_action_end_msg 1
     }
 
     succ_msg() {
-        echo " ...done."
+        log_action_end_msg 0
     }
 
     begin() {
-        echo -n $1
+       log_action_begin_msg $1 
     }
 fi
 
